nym-node/src/node/mod.rs

```diff
@@ -447,6 +448,10 @@ impl NymNode {
             wireguard: Some(wireguard_data),
             config,
             accepted_operator_terms_and_conditions: false,
+            shutdown_manager: ShutdownManager::new("NymNode")
```

Comment:
Is there a reason why this one PascalCase while the others are kebab-case?

Reply:
I'm not sure I've been particularly consistent, but I think with recent changes I attempted to use PascalCase for the top level name and kebab-case or snake_case for any child components